config/jobs/kubeflow/kubeflow-presubmits.yaml

```diff
@@ -83,6 +83,19 @@ presubmits:
       testgrid-dashboards: sig-big-data
       description: Presubmits for kubeflow/website.
       testgrid-num-columns-recent: '30'
+  - name: kubeflow-website-link-check
+    cluster: kubeflow
+    always_run: true         # Run for every PR, or only when requested.
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: ktbartholomew/link-checker:latest
```

Comment:
Where is this image coming from?
Can we pin the image so we don't break suddenly if the latest image is updated?

Reply:
Fixed.